cron/rollover_index_aliases.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# This is a temporary fix only
+
+if ! [ -x "$(command -v curl)" ]; then
+  exit 1
+fi
+
+alias_exists=`curl -s -XGET mozdef.private.scl3.mozilla.com:9200/_cat/aliases | grep "events\s"`
+events_index=`curl -s -XGET mozdef.private.scl3.mozilla.com:9200/_cat/events | grep "events\s"`
+rotate_alias=`curl -s -XGET mozdef.private.scl3.mozilla.com:9200/_cat/events | grep "events\s"`
+index_current_date=`date +%Y%m%d`
+index_new_date=$((index_current_date+1))
+index_old_date=$((index_current_date-1))
+successful_connect=`curl -s -XGET mozdef.private.scl3.mozilla.com:9200`
+
+if [[ "${successful_connect}" == *"name"* ]]; then
```

Comment:
probably should use:
```
if [[ $? -eq 0 ]]; then
...
```
because it's more reliable and probably an important check
alternative:
```
curl -s -XGET mozdef.private.scl3.mozilla.com:9200` || {
   echo "curl connection failed"
   exit 127
}
```

Reply:
but otherwise, really, you want that for each command above instead (so that you know all commands worked)